frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/general-tab/simulated-region-overview-general-tab.component.html

```diff
@@ -1,13 +1,157 @@
+<div class="h-100 mh-100 overflow-auto">
+    <div class="form-group pb-3">
+        <label class="form-label">Name</label>
+        <input
+            #nameInput="ngModel"
+            [ngModel]="simulatedRegion.name"
+            (appSaveOnTyping)="renameSimulatedRegion($event)"
+            required
+            type="text"
+            class="form-control"
+            data-cy="simulatedRegionPopupNameInput"
+        />
+        <app-display-validation [ngModelInput]="nameInput" />
+    </div>
+
+    <h6>Aktuelle Daten</h6>
+
+    <table class="table">
+        <tbody>
+            <tr
+                (click)="patientsCollapsed = !patientsCollapsed"
+                style="cursor: pointer"
+            >
+                <th scope="rowgroup" class="w-75">
+                    <i
+                        *ngIf="patientsCollapsed; else patientsExpanded"
+                        class="bi bi-caret-right-fill"
+                    ></i>
+                    <ng-template #patientsExpanded
+                        ><i class="bi bi-caret-down-fill"></i
+                    ></ng-template>
+                    Patienten
+                </th>
+                <th class="w-25">{{ (patients.all$ | async)?.length }}</th>
+            </tr>
+            <tr #collapse="ngbCollapse" [(ngbCollapse)]="patientsCollapsed">
+                <td class="ps-4">
+                    <span
+                        class="badge rounded-pill"
+                        style="background-color: red"
+                        >SK I</span
+                    >
+                </td>
+                <td>{{ (patients.red$ | async)?.length }}</td>
+            </tr>
+            <tr #collapse="ngbCollapse" [(ngbCollapse)]="patientsCollapsed">
+                <td class="ps-4">
+                    <span
+                        class="badge rounded-pill text-dark"
+                        style="background-color: yellow"
+                        >SK II</span
+                    >
+                </td>
+                <td>{{ (patients.yellow$ | async)?.length }}</td>
+            </tr>
+            <tr #collapse="ngbCollapse" [(ngbCollapse)]="patientsCollapsed">
+                <td class="ps-4">
+                    <span
+                        class="badge rounded-pill"
+                        style="background-color: green"
+                        >SK III</span
+                    >
+                </td>
+                <td>
+                    {{ (patients.green$ | async)?.length }}
+                </td>
+            </tr>
+            <tr #collapse="ngbCollapse" [(ngbCollapse)]="patientsCollapsed">
+                <td class="ps-4">
+                    <span
+                        class="badge rounded-pill"
+                        style="background-color: black"
+                        >ex</span
+                    >
+                </td>
+                <td>{{ (patients.black$ | async)?.length }}</td>
+            </tr>
```

Comment:
```suggestion
            <tr *ngFor="let patientCategoryKey of patientCategoriesMap | keys" #collapse="ngbCollapse" [(ngbCollapse)]="patientsCollapsed">
                <td class="ps-4"><app-patient-status-badge [status]="patientCategoriesMap[patientCategoryKey]" /></td>
                <td>{{ (patients[patientCategoryKey] | async)?.length }}</td>
            </tr>
```
and add this to the components class:
```ts
    public readonly patientCategoriesMap = {
        red$: 'red' as PatientStatus,
        yellow$: 'yellow' as PatientStatus,
        green$: 'green' as PatientStatus,
        black$: 'black' as PatientStatus,
    }
```

Reply:
I've chosen a slightly different implementation to avoid using the `patientCategoryMap` which effectively does not do anything but reverting what `patientCategories` and `patients` do: https://github.com/hpi-sam/digital-fuesim-manv/pull/721/commits/84701fed2df455dbb751c88073c181bb82f0d189